src/lib/ibcclient.ts

```diff
@@ -1110,6 +1163,7 @@ export class IbcClient {
     timeoutBlock?: number,
     timeoutTime?: number
   ): Promise<MsgResult> {
+    this.logger.verbose(`Transfer tokens to ${receiver}`);
```

Comment:
Is there a nice stringify method for Coin?

`Transfer ${prettyToken(token)} to ${receiver}`

Reply:
Actually, this will only be used in tests likely, so no big deal